guide/daily-spending-wallet/backup-and-recovery/manual-backup.md

```diff
@@ -134,179 +109,210 @@ Guides and gives users actionable items on how to safely **handle** their recove
 
-The goal is to encourage users to write down their recovery phrase on a physical piece of paper. You can discourage users from taking a screenshot by showing a warning modal or disabling the functionality altogether.
+The goal of the following flow is to encourage users to write down their recovery phrase on a physical piece of paper. This app disables screenshots on any screens that display a recovery phrase and, to be safe, also warns the user against screenshotting or photographing their recovery phrase.
 
 {% include /tip/open.html label="Tip: Be Clear about Numbering" icon="info" color="blue" %}
 
-Regardless of the option you decide to run with, it’s important that you explicitly instruct users to number each word (i.e. 1. sand 2. purple 3. flower). Typically, a manual recovery phrase backup scheme will ask users to recount, for example, the fifth word. It’s a bit of a pain for users to count which word corresponds to a particular number if they are not numbered initially.
+Note that this wallet explicitly instructs users to number each word (e.g. 1. sand 2. purple 3. flower). This is important because they will be asked to confirm their recovery phrase in a later step. It’s a bit of a pain for users to count which word corresponds to a particular number if they are not numbered initially.
 
 {% include /tip/close.html %}
 
-<div class="center" markdown="1">
+<div class="image-slide-gallery">
 
-{% include image.html
-   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/copy-recovery-phrase-single-word.png"
-   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/copy-recovery-phrase-single-word@2x.png"
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-choice.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-choice@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-choice@2x.png"
+   layout = "shadow"
+   caption = "Wallet prompts the user to setup a backup, and the user selects the manual backup toggle."
+   alt-text = "Screen prompting the user to back up after they have received a payment, with a manual backup toggle at the bottom"
    width = 250
-   height = 600
-   alt-text = "Showing recovery phrase one word at a time"
-   caption = "One by One UI from the [UI Kit](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=1230%3A0)"
-   layout = "float-right-desktop -background -shadow"
+   height = 541
 %}
 
-**Option 1: One by One**
-
-In a one by one design, users are given their recovery phrase one word at a time. Users are then prompted to swipe/click through each word to complete their viewing.
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-notes.png"
```

Comment:
The modal content is so big, maybe it should have it's own dedicated screen. Might need it anyways on smaller phones.

Reply:
Done